src/common/bitfield.h

```diff
@@ -0,0 +1,248 @@
+/*!
+ * Copyright 2019 by Contributors
+ * \file bitfield.h
```

Comment:
Is there any significant change from `bitfield.cuh`?

Reply:
@hcho3 No.  And this file is thoroughly tested so it should be fine.